Configuration/PyReleaseValidation/python/relval_2023.py

```diff
@@ -25,5 +25,6 @@
 numWFIB.extend([28634.0]) #2023D40
 numWFIB.extend([29034.0]) #2023D41
 numWFIB.extend([29434.0]) #2023D42
+numWFIB.extend([29834.0]) #2023D43
```

Comment:
@bsunanda I repeat my question: do you need this scenario to be routinely run in every PR test? I understood that having only the upgrade set with ```--what upgrade``` was enough...

Reply:
The above change will just run it in IB tests, not PR tests (not added to short matrix)

Reply:
you are right, this is not updating the limited selection in runTheMatrix.py https://cmssdt.cern.ch/lxr/source/Configuration/PyReleaseValidation/scripts/runTheMatrix.py#0081 , so this will just enter the standard list. One more workflow is not a big deal, I withdraw my question.